kale/task.py

```diff
@@ -76,6 +90,10 @@ def __init__(self, message_body=None, *args, **kwargs):
 
         self._dequeued()
 
+        if settings.is_multiprocess_mode() and not Task._task_process_pool:
```

Comment:
I'd rather call it "forking" mode than "master" mode, as it's more descriptive. In fairness to "multiprocessing" mode, this does use the "multiprocessing" Python library.